app/schema/answer.py

```diff
@@ -15,3 +14,45 @@ def __init__(self):
         self.templatable_properties = []
         self.options = []
         self.alias = None
+
+    @staticmethod
+    def get_instance(answer_type):
+        # Do the imports here to avoid circular dependencies
+        # TODO: Refactor out into a factory of some sort
+        from app.schema.questionnaire import QuestionnaireException
+
+        from app.schema.answers.checkbox_answer import CheckboxAnswer
+        from app.schema.answers.currency_answer import CurrencyAnswer
+        from app.schema.answers.date_answer import DateAnswer
+        from app.schema.answers.integer_answer import IntegerAnswer
+        from app.schema.answers.percentage_answer import PercentageAnswer
+        from app.schema.answers.positiveinteger_answer import PositiveIntegerAnswer
+        from app.schema.answers.radio_answer import RadioAnswer
+        from app.schema.answers.textarea_answer import TextareaAnswer
+        from app.schema.answers.textfield_answer import TextfieldAnswer
+
+        answer_type = answer_type.upper()
+
+        if answer_type == 'CHECKBOX':
```

Comment:
If we implemented the factory we could avoid all these elif block by implementing a register/create pattern

For example:

```
class AnswerFactory(object):
    def __init(self):
         self.classes = {}
    def register(self, name, class):
         self.classes[name] = class
   
    def create(name):
        return self.classes[name]()
```
Then register all the types like this:
```
answer_factory.register("CURRENCY", CurrencyAnswer)
```